qsimcirq_tests/qsimcirq_test.py

```diff
@@ -83,6 +84,38 @@ def test_cirq_qsim_simulate_fullstate(self):
     assert cirq.linalg.allclose_up_to_global_phase(
         result.state_vector(), cirq_result.state_vector())
 
+  def test_cirq_qsim_simulate_random_unitary(self):
+
+    q0, q1 = cirq.LineQubit.range(2)
+    qsimSim = qsimcirq.QSimSimulator(qsim_options={'t': 16, 'v': 0})
+    for _ in range(10):
+        random_circuit = cirq.testing.random_circuit(qubits=[q0, q1],
```

Comment:
I support the notion of fuzz-testing against random circuits, but to avoid flakiness this should specify a seed (e.g. the iteration number) in its `random_state` argument.

Reply:
Good point. I have added using the iteration number as the random_state argument for random_circuit.